.azure/pipelines/ci.yml

```diff
@@ -584,6 +584,17 @@ stages:
         chmod +x $HOME/bin/jq
         echo "##vso[task.prependpath]$HOME/bin"
       displayName: Install jq
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core sdk'
+      inputs:
+        packageType: sdk
+        # The SDK version selected here is intentionally supposed to use the latest release
+        # For the purpose of building Linux distros, we can't depend on features of the SDK
+        # which may not exist in pre-built versions of the SDK
+        # Pinning to preview 8 since preview 9 has breaking changes
+        version: 3.1.100
+        installationPath: $(DotNetCoreSdkDir)
+        includePreviewVersions: true
```

Comment:
This bit was intentionally removed in 8b7f6621695d93b2a55fb8a5b1be99c4af867ae4. If anything, that fix should be ported to 'release/3.0'.

Reply:
Let's file an issue for this.  Not doing i this way means that the source-build legs must download the extra dotnet runtimes. This means feeding more internal runtime download parameters to various places. I'm skeptical the source build leg should be doing this at all.